tiledb/sm/rest/rest_client.cc

```diff
@@ -511,6 +512,59 @@ Status RestClient::post_array_metadata_to_rest(
       stats_, url, serialization_type_, &serialized, &returned_data, cache_key);
 }
 
+std::vector<shared_ptr<const Enumeration>>
+RestClient::get_enumerations_from_rest(
+    const URI& uri,
+    uint64_t timestamp_start,
+    uint64_t timestamp_end,
+    Array* array,
+    const std::vector<std::string>& enumeration_names) {
+  if (array == nullptr) {
+    throw Status_RestError(
+        "Error getting enumerations from REST; array is null.");
+  }
+
+  Buffer buf;
+  serialization::serialize_load_enumerations_request(
+      array->config(), enumeration_names, serialization_type_, buf);
+
+  // Wrap in a list
+  BufferList serialized;
+  throw_if_not_ok(serialized.add_buffer(std::move(buf)));
+
+  // Init curl and form the URL
+  Curl curlc(logger_);
+  std::string array_ns, array_uri;
+  throw_if_not_ok(uri.get_rest_components(&array_ns, &array_uri));
+  const std::string cache_key = array_ns + ":" + array_uri;
+  throw_if_not_ok(
+      curlc.init(config_, extra_headers_, &redirect_meta_, &redirect_mtx_));
+  const std::string url = redirect_uri(cache_key) + "/vXKCD/arrays/" +
+                          array_ns + "/" + curlc.url_escape(array_uri) +
+                          "/get_enumerations?" +
```

Comment:
How about ``enumerations` instead of `get_enumerations`

Reply:
Done.